mastering-plone/dexterity_2.rst

```diff
@@ -615,7 +615,7 @@ Add a new file :file:`profiles/default/registry.xml`
 Add versioning through GenericSetup
 ------------------------------------
 
-Configure the versioning policy and a diff-view for talks through GenericSetup.
+Configure the versioning policy and a diff view for talks through GenericSetup.
```

Comment:
Is GenericSetup an inline literal? If so, please surround with double backticks.